posts/posts.csv

```diff
@@ -1,17 +1,20 @@
+Reviewer perspectives;posts tagged reviewer;https://ropensci.org/tags/reviewer/;
 Contributing to or taking over maintenance of a package;Relaunching the qualtRics package;https://ropensci.org/blog/2019/04/30/qualtrics-relaunch/;has an engaging tone while being informative and providing technical details.
 Contributing to or taking over maintenance of a package;Monkeying around with Code and Paying it Forward;https://ropensci.org/blog/2018/04/20/monkeydo/;gives insight into how the author is thinking about workflows and contributing to rOpenSci.
 Summarizing a Community Call;Code Review in the Lab;https://ropensci.org/blog/2018/11/29/codereview/;
-Describing a creative use case for multiple rOpenSci packages; Where to go observe birds in Radolfzell? An answer with R and open data;https://ropensci.org/blog/2018/08/14/where-to-bird/;
-Using an rOpenSci resource, such as the Package Development Guide or a task view;When Standards Go Wild - Software Review for a Manuscript;https://ropensci.org/blog/2019/04/18/wild-standards/;
-Using an rOpenSci resource, such as the Package Development Guide or a task view;Using rOpenSci Software Peer Review Guidelines for Teaching;https://ropensci.org/blog/2019/08/27/software-peer-review-guidelines-for-teaching/;
-Using an rOpenSci resource, such as the Package Development Guide or a task view;Getting your toes wet in R: Hydrology, meteorology, and more;https://ropensci.org/blog/2019/04/02/hydrology-task-view/;
+Using an rOpenSci resource, such as the Dev Guide or a task view;When Standards Go Wild - Software Review for a Manuscript;https://ropensci.org/blog/2019/04/18/wild-standards/;
+Using an rOpenSci resource, such as the Dev Guide or a task view;Using rOpenSci Software Peer Review Guidelines for Teaching;https://ropensci.org/blog/2019/08/27/software-peer-review-guidelines-for-teaching/;
+Using an rOpenSci resource, such as the Dev Guide or a task view;Getting your toes wet in R: Hydrology, meteorology, and more;https://ropensci.org/blog/2019/04/02/hydrology-task-view/;
+About a peer-reviewed package developed by the authors;rmangal: Making Ecological Networks Easily Accessible;https://ropensci.org/blog/2019/10/21/rmangal/;talks through the scientific problem and context, shows some code examples, and talks about peer review but doesn't make that the dominant part of the post.
+About a peer-reviewed package developed by the authors;Forcing Yourself to Make Your Life Easier;https://ropensci.org/blog/2018/04/12/ijtiff/;is an honest post with some reflection and an important message.
+About a peer-reviewed package developed by the authors;The av Package: Production Quality Video in R;https://ropensci.org/technotes/2018/10/06/av-release/;(tech note) is to the point. av is not a peer-reviewed package but this is a good example of a tech note.
+Major updates to key packages;(Re)introducing skimr v2 - A year in the life of an open source R project;https://ropensci.org/blog/2019/10/29/skimrv2/);
```

Comment:
Should we rephrase "key" to something like "generalist"?

Reply:
Hmm, good point, what about "popular"? That's a bit weird though, "important" isn't much better. 

Perhaps, "Major updates to packages with broad audiences"? a bit less "are you in the popular crowd?"

Reply:
> Major updates to packages with broad audiences

I like that!